solutions/azure-automation-state-configuration/azuredeploy.bicep

```diff
@@ -0,0 +1,594 @@
+param location string = resourceGroup().location
```

Comment:
please let's add descriptions to every param

Reply:
done